Makefile

```diff
@@ -0,0 +1,2 @@
+run:
+	docker-compose up --build
```

Comment:
Given that the `docker-compose` setup was requested by Windows users, and that Windows users don't have `make`, I'm not sure this is 100% required, but I include it as a form of documentation.